src/main/java/net/tropicraft/Tropicraft.java

```diff
@@ -128,6 +145,66 @@ public Tropicraft() {
         });
     }
 
+    private static void setupConfig(ForgeConfigSpec.Builder builder) {
```

Comment:
Very unimportant- would it make sense to move this setup logic into the `TropicraftConfig` class?